.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,7 +32,7 @@ Note: If merging this PR should also close the associated Issue, please also add
 
 #### EIA/NEURON Testing Checklist
 * When creating a PR:
-- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true` or `neuron_mode = true`
+- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`
```

Comment:
If you're going to mention `graviton_mode`, you should update the section header to `EIA/NEURON/GRAVITON Testing Checklist`

Reply:
will change it